### DISCUSSION TITLE
HIP-14: Hold

### DISCUSSION THREAD

**Discussion by kenthejr** [hiero-ledger/hiero-improvement-proposals#59] [Edits]
This discussion is designated as the `discussions-to` target of [hip-14](https://github.com/hashgraph/hedera-improvement-proposal/blob/master/HIP/hip-14.md). Use this thread to further develop the HIP and move it through to its final lifecycle state.

**Comment by topshef** [comment]
This sounds interesting. Is this somewhat already possible with a combination of compliance keys (eg freeze) and scheduled transactions?

**Comment by Ferparishuertas** [comment]
Thanks @topshef for the feedback.
Correct me if I am wrong... Freeze "locks" all th account and does not allow amount freeze isnt it?

Hold is an allowance with linked locked amount.

**Reply by topshef** [reply]
Ah yes that's right, freeze would lock the whole lot. I guess a workaround without this HIP might be to manage 2 accounts, one for available and one for held. Sorry haven't thought it through though, so forgive the brain f@rt !

**Comment by sergmetelin** [comment]
@Ferparishuertas can we say this functionality is covered by https://hips.hedera.com/hip/hip-24?

**Comment by sergmetelin** [comment]
@Ferparishuertas Assigning status "Superseded by" HIP-24 to this HIP.

**Comment by Ferparishuertas** [comment]
Pausing and holding is not the same @serg. Hold is "locking" tokens for a
given account, and pausing  is pausing the whole token,

Maybe I am wrong but I am not seeing id superseeded

El mar, 30 nov 2021 a las 21:59, Serg Metelin ***@***.***>)
escribió:

> @Ferparishuertas <https://github.com/Ferparishuertas> Assigning status
> "Superseded by" HIP-24 to this HIP.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/hashgraph/hedera-improvement-proposal/discussions/59#discussioncomment-1724724>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAKXFDWR6KZHZWFW2TCWF33UOU3JNANCNFSM4ZYCKV2A>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>


-- 
[image: logo] <https://io.builders/>
*io.builders <http://io.builders/>*
Blockchain venture builder *Fernando París*
CTO • ioBUILDERS
Mobile: +34 679 467 927
Parque Científico y Tecnológico UPM Campus de Montegancedo, s/n.
28223 Pozuelo de Alarcón, MADRID


NOTA LEGAL: Este mensaje y cualquier archivo adjunto está destinado
únicamente a quien se dirige y es confidencial. Si usted ha recibido este
mensaje por error, comuníqueselo al remitente y bórrelo inmediatamente. La
utilización, revelación y/o reproducción del mensaje puede constituir un
delito.

PROTECCIÓN DE DATOS – Responsable: ioBUILDERS, S.L., Finalidad. Envío de
información, respuesta a consultas y contactos genéricos, mientras dure
nuestra relación y tengamos su consentimiento. Destinatarios. No se cederán
datos a terceros salvo obligación legal. Derechos. Puede ejercer los
derechos de acceso, rectificación, supresión y oposición, limitar el
tratamiento de sus datos, o directamente oponerse al tratamiento, o ejercer
el derecho a la portabilidad de los mismos. Todo ello, mediante escrito,
acompañado de copia de documento oficial que le identifique, dirigido a
Parque Científico y Tecnológico UPM Campus de Montegancedo, s/n. 28223
Pozuelo de Alarcón, MADRID (España). En caso de disconformidad con el
tratamiento, también tiene derecho a presentar una reclamación ante la
Agencia Española de Protección de Datos. También podrá oponerse a nuestros
envíos de comunicaciones comerciales (Art.21.2 de la LSSI) a través de la
siguiente dirección de correo electrónico: ***@***.***
